### PR TITLE
Added Remove Built-in Apps FAQ doc.

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

File: content/en/docs/faq/applications/remove-built-in-apps.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Remove the Built-in Applications in KubeSphere"
+keywords: "KubeSphere, OpenPitrix, Application, App"
+description: "How to remove the built-in applications in KubeSphere"
+linkTitle: "Remove Built-in Apps in KubeSphere"
+Weight: 16910
+---
+
+As an open-source and app-centric container platform, KubeSphere integrates 15 built-in apps in the App Store that is based on [OpenPitrix](https://github.com/openpitrix/openpitrix).
```
Comment (Sherlock113) [inline content/en/docs/faq/applications/remove-built-in-apps.md:9]: Add the following sentence here:

They are accessible to all tenants in a workspace, while you can also remove them from the App Store. This tutorial demonstrates how to remove a built-in app from the App Store.

File: content/en/docs/faq/applications/remove-built-in-apps.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+If you want to remove the built-in apps in KubeSphere, you have to use the account `admin` with the default role of `platform-admin` to log in to the web console of KubeSphere, and then follow the steps as below:
+
+1. Click **Platform** in the upper left corner, and then select **App Store Management**.
```
Comment (Sherlock113) [inline content/en/docs/faq/applications/remove-built-in-apps.md:13]: 1. Click **Platform** in the upper left corner, and then select **App Store Management**.
->
1. Log in to the web console of KubeSphere as `admin`, click **Platform** in the upper left corner, and then select **App Store Management**.

File: content/en/docs/faq/applications/remove-built-in-apps.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+   ![click-platform](/images/docs/zh-cn/faq/applications/remove-built-in-apps/click-platform.PNG)
+
+   ![select-app-store-management](/images/docs/zh-cn/faq/applications/remove-built-in-apps/select-app-store-management.PNG)
+
+2. In the **App Store** page, you can see all 15 built-in apps displayed in the list. Click **tomcat** to go to its detail page.
```
Comment (Sherlock113) [inline content/en/docs/faq/applications/remove-built-in-apps.md:19]: Click **tomcat** to go to its detail page.
->
Select an app that you want to remove from the App Store. For example, click **tomcat** to go to its detail page.

File: content/en/docs/faq/applications/remove-built-in-apps.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+---
+title: "Remove the Built-in Applications in KubeSphere"
```
Comment (Sherlock113) [inline content/en/docs/faq/applications/remove-built-in-apps.md:2]: Remove the Built-in Applications
->
Remove Built-in Apps

Note that, in most cases, we need to make sure the value of `title` and `linkTitle` are the same.

File: content/en/docs/faq/applications/remove-built-in-apps.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+keywords: "KubeSphere, OpenPitrix, Application, App"
+description: "How to remove the built-in applications in KubeSphere"
```
Comment (Sherlock113) [inline content/en/docs/faq/applications/remove-built-in-apps.md:4]: remove the built-in applications
->
remove built-in apps

**Review by Sherlock113 — COMMENTED** [review]
Have a look at my comments and make changes. Make sure you change the English source file in the zh folder as well.

File: content/en/docs/faq/applications/remove-built-in-apps.md (hunk 4 above)
Comment (Felixnoo) [reply content/en/docs/faq/applications/remove-built-in-apps.md]: Noted.

**Pull request merged**